docs/vercel/src/Index.svelte

```diff
@@ -61,14 +82,36 @@
     Suggestions and PRs are welcome at <a
       href="https://github.com/o2sh/onefetch">github.com/o2sh/onefetch</a>
   </p>
-  <h3>Languages <small>({languages.length})</small></h3>
-  {#each languages as language}
+  <div class="title">
+    <h3>Languages <small>({$filteredLanguages.length})</small></h3>
+    <button class="filter-button" on:click={() => (menuOpen = !menuOpen)}
+      >Filter by type</button>
+  </div>
+
+  <div class:show={!menuOpen} class="checkbox-group">
+    {#each languageTypes as type}
+      <label for={type}>
+        <input
+          id={type}
+          type="checkbox"
+          value={type}
+          bind:group={$filter.checkboxes} />
+        {type}
+      </label>
+    {/each}
```

Comment:
TBH I wonder if the filters really need to be shown/hidden, since they don't take up much space.

Reply:
done